source/_integrations/tibber.markdown

```diff
@@ -55,6 +55,37 @@ The `tibber` sensor provides the current electricity price if you are a [Tibber]
 You also get sensors for monthly consumption, monthly cost, and monthly peak hour
 If you have a Tibber Pulse it will also show the electricity consumption in real time.
 
+## Available Sensors
+
+- Accumulated cost since midnight (requires active Tibber power deal)
+- kWh consumed since midnight
+- net kWh produced since midnight
+- net kWh produced since last hour shift
+- Current on L1, L2 and L3
+- Estimate of kWh consumption current hour
+- kWh consumed since since last hour shift
+- Average consumption since midnight (Watt)
+- Last meter active import register state (kWh)
+- Last meter active export register state (kWh)
+- Peak consumption since midnight (Watt)
+- Min consumption since midnight (Watt)
+- Consumption at the moment (Watt)
+- Consumption at the moment (Watt)
+- Net production (A-) at the moment (Watt)
+- The total price (energy + taxes)
+- Device signal strength (Pulse - dB; Watty - percent)
+- Voltage on phase 1, 2 and 3
+
+<div class='note'>
+
+Sensors for information below are created but suspected to be deprecated from Tibbers side since they aren't populated
+
+- Monthly cost
+- Monthly net consumption
+- Monthly peak hour
+- Time of max hour consumption
+
+</div>
```

Comment:
We should not add this unless we are sure. (and thus not suspecting).

In case they are deprecated, we should remove them from the codebase. A comment in the docs will not clean them up or fix an issue.